src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -38,13 +38,12 @@
 
 (defview quoted-message
   [_ {:keys [from text]} outgoing current-public-key]
-  (letsubs [{:keys [ens-name alias]} [:contacts/contact-name-by-identity from]]
+  (letsubs [contact-name [:contacts/contact-name-by-identity from]]
```

Comment:
probably not in this PR but we should add this data to the message itself in the event, because of https://github.com/status-im/status-react/issues/10380

Reply:
I would say it's even better to do not update the name when you on the chat view, and update it only next time you open chat, because 1- its rare case when this happens, 2 - we have 30s delay for verifications, so from performance and from ux it will be better to show it next time you open chat

Reply:
Most of the times though it does happen when you are in the chat view, think of the simplest case:
1) Open #status
2) Receive messages, no names, status-go verifies them
3) Names are received

In such case I'd want to see a name as soon as it's available, not on reloading the chat?

We can though look at performance, my guess is that this is not the bottleneck, more likely is the actually building up the list (datemarks,timestamps etc)